testflight_invite.py

```diff
@@ -150,33 +198,35 @@ def numTesters(self):
 
     def addTester(self, email, firstname='', lastname=''):
         self.login()
+        params = { 'email': email, 'firstName': firstname, 'lastName': lastname }
+        try:
+            text = self.readData(
+                'https://itunesconnect.apple.com/testflight/v2/providers/{teamId}/apps/{appId}/testers'.format(
+                    teamId=self.contentProviderId, appId=self.appId
+                ),
+                json.dumps(params),
+                headers={'Content-Type': 'application/json'}
+            )
+        except urllib2.HTTPError as e:
+            raise
+
+        testerResponseBody = json.loads(text)
+        testerId = testerResponseBody['data']['id']
         params = {
-            'users': [
-                {
-                    'emailAddress': {
-                        'value': email
-                    },
-                    'firstName': {
-                        'value': firstname
-                    },
-                    'lastName': {
-                        'value': lastname
-                    },
-                    'testing': {
-                        'value': 'true'
-                    }
-                }
-            ]
+            'groupId': self.groupId,
+            'testerId': testerId,
         }
+
         try:
             return self.readData(
-                'https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/user/externalTesters/%s/' % self.appId,
+                'https://itunesconnect.apple.com/testflight/v2/providers/{teamId}/apps/{appId}/groups/{groupId}/testers/{testerId}'.format(
+                    teamId=self.contentProviderId, appId=self.appId, groupId=self.groupId, testerId=testerId
+                ),
                 json.dumps(params),
-                headers={'Content-Type': 'application/json'}
+                headers={'Content-Type': 'application/json'},
+                method='PUT'
             )
         except urllib2.HTTPError as e:
-            if e.code == 500: # 500 if tester already exists... This is not how you HTTP, Apple.
-                raise TFInviteDuplicateException
             raise
```

Comment:
Same comment as above with respect to just allowing calling function to handle errors since we're just raising here.